core/include/detray/coordinates/coordinate_base.hpp

```diff
@@ -220,107 +221,28 @@ struct coordinate_base {
         const stepper_state_t& stepping, const transform3_t& trf3,
         const mask_t& mask) const {
 
-        if constexpr (stepper_state_t::id != stepping::id::e_linear) {
-            return path_correction(stepping(), trf3, mask,
-                                   stepping._step_data.b_last);
-        } else {
-            return path_correction(stepping(), trf3, mask);
-        }
+        return this->path_correction(stepping().pos(), stepping().dir(),
```

Comment:
Can't we call this interface directly?

Reply:
You mean without `this`?

Reply:
or you meant just removing the wrapper function

Reply:
Ah, sorry, I meant remove the wrapper function